### ISSUE TITLE
Add HP Gen10 Support with ILO5

### ISSUE THREAD

**Issue by FLiPp3r90** [CSCfi/ansible-role-hp-firmware-upgrade#23]
HP Gen10 with ILO5 isn't supported. This should be solved :)

**Comment by martbhell** [comment]
Indeed it should! I don't have any iLO5 hosts to play with.
Are you saying iLO5 does not work? Any errors? Is the PCI address different?

**Comment by FLiPp3r90** [comment]
Hi @martbhell,

there are a restriction in the ilo.yml that assert ILO4.

`- assert: { that: ilo_version is match("ilo4") }`

I've find another way to determine the ILO version without getting the PCI address:

```  
     - name: Checking for ILO device
      shell: hponcfg -w conf.txt|grep -i 'Firmware Revision'| awk '{print $8$9}'
      register: ilo_info
      changed_when: False
      check_mode: no 
```

I think, this is a better way and it will work  for ILO5. But I can't test it, because we havn't any Gen10 servers.

**Comment by martbhell** [comment]
Cool. Let's see what we can find :)

**Comment by FLiPp3r90** [comment]
Fix suggestion -> Pull Request https://github.com/CSCfi/ansible-role-hp-firmware-upgrade/pull/24

**Comment by Jukepoika** [comment]
This is now fixed with PR #33.
